src/util/Permissions.js

```diff
@@ -32,14 +32,28 @@ class Permissions {
 
   /**
    * Gets all given permissions that are missing from the bitfield.
-   * @param {PermissionResolvable[]} permissions Permissions to check for
+   * @param {PermissionResolvable[]|PermissionResolvable} permissions Permission(s) to check for
    * @param {boolean} [checkAdmin=true] Whether to allow the administrator permission to override
    * @returns {PermissionResolvable[]}
    */
   missing(permissions, checkAdmin = true) {
+    if (permissions instanceof PermissionResolvable) return this.missing(Permissions.resolveToObject(permissions).toArray(), checkAdmin);
```

Comment:
`PermissionResolvable` isn't a class

Reply:
Is there a way to check whether or not a variable is a `PermissionResolvable`?

Reply:
This line needs to be completely reverted to what you had before.

Reply:
Alright, I was just hoping that a `PermissionResolvable` could be accepted instead of an instance of `Permissions`. If there's no way to do that, I'll put it back.

Reply:
what if we return the array using `Array.filter()` `if (permissions instanceof Array)`? That way it will be a `PermissionResolvable` otherwise and we could call `return this.missing(new Permissions(permissions).toArray())`.

Reply:
should I push a commit with it so you can see?

Reply:
I did, check the latest one